CMakeLists.txt

```diff
@@ -20,14 +20,16 @@ if(CGLM_USE_C99)
   set(C_STANDARD 99)
 endif()
 
-if("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
-  list(APPEND LDFLAGS "/W4")
-elseif("${CMAKE_C_COMPILER_ID}" STREQUAL "GNU")
-  set(C_EXTENSIONS YES)
-  list(APPEND LDFLAGS "-Wall" "-Wpedantic" "-Wstrict-aliasing=2")
-  list(APPEND LDFLAGS "-Werror=strict-prototypes" "-fstrict-aliasing" "-O3")
-elseif("${CMAKE_C_COMPILER_ID}" STREQUAL "CLANG")
-  list(APPEND LDFLAGS "-Wall" "-Weverything" "-Wpedantic")
+if(MSVC)
+	add_definitions(-DNDEBUG -D_WINDOWS -D_USRDLL -DCGLM_EXPORTS -DCGLM_DLL)
+	add_compile_options("/W3" "/Ox" "/Gy" "/Oi" "/TC" "/analyze")
+else()
+    add_compile_options("-Wall" "-Werror" "-O3")
+    if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
+		add_compile_options("-stdlib=libc++" "-Weverything")
+    else()
+		add_compile_options("-Werror=strict-prototypes" "-fstrict-aliasing")
```

Comment:
I think we must add `-fstrict-aliasing` to clang too?

Reply:
Yes, again, my bad!

Reply:
@recp 
In ```clang```:
According to [this page](http://www.keil.com/support/man/docs/armclang_ref/armclang_ref_sam1465825195202.htm), the compiler automatically adds ```-fstrict-aliasing``` when the ```-O1``` or above is specified, making the specifying of the option redundant.

In ```gcc```:
According to [this page](https://gcc.gnu.org/onlinedocs/gcc/Optimize-Options.html), the compiler automatically adds ```-fstrict-aliasing``` when the ```-O2``` or above is specified, making the specifying of the option redundant again.

I will remove the redundant options. Thought of giving you a heads-up!